src/metadata.js

```diff
@@ -43,23 +43,37 @@ function appendMetadata(docs) {
 }
 
 function evalClassification(label) {
+  // const wordsArray = label.split(' ')
+  // const firstThreeWords = `${wordsArray[0]} ${wordsArray[1]} ${wordsArray[2]}`
   if (
     // Here for unknown reason, sometimes we find "Avis de dégrèvement" and sometimes "Avis dégrèvement"
     // We have to cover both possibilities
     label.match(
-      /Avis (de situation|de taxes?|d'impôt|de dégrèvement|dégrèvement|supplémentaire)/
+      /Avis (de situation|de taxes?|d'impôt|de dégrèvement|dégrèvement|supplémentaire|impôts sur)/
```

Comment:
Seems a duplicate with L55 to me, no ?
'Avis impôts sur'

Reply:
right, eagle eye :+1: